.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+name: Dependabot Auto Approve
```

Comment:
I *think* this file can be removed with the version 2 dependabot (and the merge me task). I'm still waiting to get confirmation on this on one of our other repos (waiting for a dependabot change to come through).